src/enchant.h

```diff
@@ -52,7 +52,7 @@ typedef struct str_enchant_broker EnchantBroker;
 typedef struct str_enchant_dict   EnchantDict;
 
 /* const */
-ENCHANT_MODULE_EXPORT (char *)
+ENCHANT_MODULE_EXPORT (const char *)
```

Comment:
One thing I'm sure is that it was invalid. gcc warnings should already have complained.